src/main/java/entities/fridge/CommonFridge.java

```diff
@@ -13,13 +12,13 @@ public class CommonFridge implements Fridge, Serializable {
      * User can make changes to the ingredients list by adding or removing an ingredient.
      * Existing user can set their Ingredient to their previous ingredients list.
      */
-    private ArrayList<CommonIngredient> ingredients;
+    private ArrayList<Ingredient> ingredients;
```

Comment:
Make sure that the rest of the team is okay with these changes from CommonIngredient to Ingredient.

Reply:
What is the reason for changing all of them from common ingredient to their interfaces?